client/pkiclient/cache.go

```diff
@@ -159,46 +159,82 @@ func (c *Cache) insertLRU(newEntry *cacheEntry) {
 }
 
 func (c *Cache) worker() {
+	const retryTime = time.Second / 2
```

Comment:
why do we use 0.5 second here? do we need to use backoff algorithm?

Reply:
This cache client will try to update the epoch every 0.5 seconds to see if there is a new epoch. Do you have other ways to check the epoch in your mind?